### DISCUSSION TITLE
Categories and Tags

### DISCUSSION THREAD

**Discussion by FredMark** [OpenLitterMap/openlittermap-web#85] [Ideas]
I like OLM in the way that it 'prescribes' the tags that you have to give.
With that i mean that there is no free textfield where you can type the tags as you want them and thus preventing from getting a lot of different tags and probably also a lot of tags that actually mean the same thing but were a result of typing mistakes like
'M & M s' of 'M&M's' or MMs etc or like 'McDonalds', 'Mc Donalds' etc.
However, it might also give some problems because, as an example, if i find some tinfoil, I have no idea where to find this tag (at least not in web, where i have to open all the categories and scroll through all the lists to see if there is a tag 'tinfoil'.
(after a while i realized that it was under food, and the tag was 'aluminium foil'.

So maybe that is the question i have.
How do we go from here?
**What are the categories that we need and what are the tags that we need.**
Or **do we also need an extra 'free text'-field** to add some tags, for instance if i want to tag a can with the brand 'Amstel', which is not in the list of brands so that one day, this can be added as an extra tag if (for instance) more than 100 pictures really have that extra text-tag?

Another example of something where i think that it might not be the optimum solution is 'plastic bottles'.
I find a lot of them and it is not always clear what the contents of the bottle used to be.
However, if i use the text/searchfield in the app and type 'bottle',
I find 'plastic water bottle' and 'plastic fizzy drink bottle'.
However, if i don't know what was in it, how do i know what to choose?
And if these are the 2 options, how about a plastic milk bottle or a plastic beer bottle or plastic Redbull bottle.
So actually, what i think, is that we should not have 2 or 3 things in 1 tag but have tags for different things.
It is not a 'plastic water bottle', it is a bottle & it is made of plastic & the contents were water.
in that way we don't need tags like:
- plastic beer bottle
- glass beer bottle
- plastic redbull bottle
- metal redbull can
- plastic water bottle
- plastic fizzy drink bottle
- (beware, we don't even have a 'glass fizzy drink bottle' so how would you tag that one at this moment?)
etc etc
But we would need some specific tags like
- plastic / glas / metal / paper (material)
- water / beer / wine / milk (content)
- bottle / cup / can /  (object)

Depending on the need, we might also need something like **the state it was in**.
In the Netherlands we are also looking at things like 'was the can still 'complete' or was it already shredded by a lawnmower?
Is the lid of the mcdonalds-cup still in one piece, or was it already degrading
Was the plastic joint-container still in one piece, or did a car run over it and was it broken into many pieces?

Another thing that might be interesting is the **volume.**
For instance for measuring how much space it would take if it was not found on the streets but if it was put in a bin. If people would all throw their litter in a bin, that would be cool, but it would also mean that bins would be much fuller all of the time, what would the consequences be.
(By the way, i am not saying that everybody should also administer these facts, but in some cases it could be meaningful if it was possible).

And of course, what i am also curious about, is **what other people would find interesting data** about the items that we capture?
If we collect data for others to use, what would those others be looking for?
**When would the data be valuable?**

**Comment by FredMark** [comment]
Another problem that i am seeing, working on the data-export.
In web and app, you have the option to tag a picture as plastic cup, and you can find this plastic cup under 'alcohol',
however, you can also find a plastic cup under 'softdrinks'.
in the export that is filled with a '1' if you tag it via 'softdrink' is called 'plastic_cups'.
However, by the name of this column, you can not see if it was a alcohol-plastic cup or a softdrink-plastic cup.

The fix for that would be to rename the column to 'alcohol-plastic cup' and 'softdrink-plastic cup'.
But do we really need 2 columns for that?

Would we also need a 'plastic cup' if we would introduce a category 'dairy' and the cup used to be filled with milk or chocolademilk'?

And what about a straw. If i find a straw, i can tag it under softdrinks>straw and it would show up in excel under the column 'straws'.
But if i should find it after a warm day on the beach, i could also find a straw under 'coastal/marine'. However, only a 'degraded straw'. So if it was not degraded, i would have to tag it as 'softdrink>straw'.
But what if i find a degraded straw, far away from the beach/coast, it wouldn't make sense to take it under 'coastal' but yet that is the only tag we have with 'degraded'.
So i think that we should also seperate the state of the item from the item itself. Have a category with items (like straw/balloon etc) and a category with the state of the item (like degraded, mowed into pieces, shredded or whatever, like mentioned in the original post) and maybe even a category with something like 'location' with values like 'in the gutter, on the beach, in the see, near the lake, in the lake etc?

**Comment by FredMark** [comment]
![Schermafdruk 2021-01-29_08 56 33 (irfanview)](https://user-images.githubusercontent.com/2518047/106276822-68bf7300-6238-11eb-8586-36b54cc36462.jpg)
![20190827_1925_IMG_4783](https://user-images.githubusercontent.com/2518047/106276891-842a7e00-6238-11eb-8cf1-dd7c53948a7d.JPG)

Just keeping notes for myself while tagging litter, what do i see that gives me a hardtime tagging:

- A box of 'Smint', some kind of peppermint. It is a metal box, so 'plastic packaging' or 'paper packaging' under food is not correct and 'metal object' under 'other' also does not really say it.
- A glass coca-cola bottle? There is nothing under drinks that would apply?
- Picture with some glass. The only 'broken glass' we have, is under 'Alcohol', but it is not always 'Alcohol'-related.
- No appropriate tag for a laughinggass-container (normally used for whippedcream, but used as 'narcotics'
- No appropriate tag for fireworks
- No appropriate tag for can with alcohol like bacardi/Jack Daniels etc. (i choose can, which is 'softdrink')
- We have 'juice carton', but i found a 'chocolademilk carton'. Why not just have a 'drinkcarton', and, if we also want to make clear what the contents were, a catergory with 'contents' with values like 'beer, milk, juice, soda, water', etc.
- We have plastic and paper packaging, but we also have 'sweet wrappers'. This last one does not say the material (and in the Netherlands we have both, plastic and paper). So what do i use if i have a paper box that contains sweet? (if i choose paper packaging, i loose the data that it is regarding sweets, but if i use sweetwrapper, i am loosing data about the material (and in the Netherlands this is an important one, because we are trying to convince companies to move from plastic to paper.

**Comment by xlcrr** [comment]
Reuseable water bottles

**Reply by xlcrr** [reply]
Brands - Monster
